core/ecschema-editing/src/Editing/Entities.ts

```diff
@@ -13,16 +13,22 @@ import {
 import { PropertyEditResults, SchemaContextEditor, SchemaItemEditResults } from "./Editor";
 import { ECClasses } from "./ECClasses";
 import { MutableEntityClass } from "./Mutable/MutableEntityClass";
+import { NavigationProperties } from "./Properties";
 
 /**
  * @alpha
  * A class extending ECClasses allowing you to create schema items of type EntityClass.
  */
 export class Entities extends ECClasses {
   public constructor(_schemaEditor: SchemaContextEditor) {
```

Comment:
This comment applies to all constructors. The schemaEditor argument starts with an underscore, which is actually the way to mark unused methods arguments, as we use it though (passing to the parent constructor), we should not have a leading underscore here.

Reply:
Done